src/object.js

```diff
@@ -43,17 +43,15 @@ _.extend(MarionetteObject.prototype, Backbone.Events, {
   //this is a noop method intended to be overridden by classes that extend from this base
   initialize: function() {},
 
-  destroy: function(options) {
+  destroy: function(...args) {
     if (this._isDestroyed) { return this; }
 
-    options = options || {};
-
-    this.triggerMethod('before:destroy', options);
+    this.triggerMethod('before:destroy', ...args);
```

Comment:
gak.. comment got outdated.. was saying we could still support this via 
```
options = options || {};
this.triggerMethod('before:destroy', options, ...args);
```
or perhaps better.. I think this should work now `destroy: function(options={}) {`

But I'm ok with the change too.. dunno why we are doing it here and not on `View#destroy`